tasks/main.yml

```diff
@@ -133,6 +133,17 @@
         _config_yml is changed or
         _sentry_conf_py is changed
 
+- name: Create Sentry users
```

Comment:
There's something in this command, probably related to parts of the command being in separate lines is causing it to fail. maybe adding a ` \` before every linebreak in the command will fix this?